portfolio/src/main/java/com/google/sps/servlets/DeleteServlet.java

```diff
@@ -38,16 +36,13 @@
    */
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    UserService userService = UserServiceFactory.getUserService();
-    if (userService.isUserAdmin()) {
```

Comment:
You should still keep this check - as Hao mentioned in the team meeting today, malicious users might try to send a POST request directly via URL and try to delete the comments, even if they don't have the `Delete` button available.

Reply:
Yep I fixed all these in another PR :)